notebooks/week3/03.model_serving_feature_lookup.py

```
@@ -0,0 +1,160 @@
+num_features = config.num_features
+cat_features = config.cat_features
+required_columns = cat_features + num_features
+required_columns
+
+# COMMAND ----------
+
+fe_columns = ["person_income"]
+for f in fe_columns:
+    required_columns.remove(f)
+
+required_columns  
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)